docs/organizer-guide/confs/start.rst

```diff
@@ -0,0 +1,42 @@
+I Want to Run a Write the Docs conference, Now What?
+====================================================
+
+We have two guidelines that you should always adhere to when thinking about
+starting your own Write the Docs event:
+
+* **You must have attended a Write the Docs conference.**
+
+  We do this to ensure that you know what a "Write the Docs event" means, for
+  better or for worse. We run these events as not-for-profit experiments where
+  we try to isolate the things that work well and remove, anything that degrades
+  the experience. We take risks where others might be cautious, we focus on
+  shared social experiences, and we do our best to make a difference in the
+  community.
+
+  Reading about this philosophy second-hand just doesn't convey the full scope of
+  how we operate.
+
+* **You must have an established Write the Docs organizer onboard for guidance.**
+
+  We run Write the Docs events as a franchise model where the local team are in
+  control. They arrange the schedule, scope out the venue, plan the parties, and
+  organize everything else. This gives the event a local feel which is part of
+  what makes it special: if you attend Write the Docs US and Write the Docs
+  Europe you will find significant differences -- and that is intentional.
+
+  We have this guideline so we can ensure that the conference is organized in
```

Comment:
I would probably reword this to something more explicit similar to:

"However, to ensure some basic consistency between the different events in philisophy and structure, first-time organizers will be assigned a mentor from our council of elders <link>, who are long-time organizers and can provide guidance and answer questions that might arise during the process."